app/templates/image-set/index.hbs

```diff
@@ -1,71 +1,96 @@
         </div>
       </div>
-    </div>
+      <div class='control-panel-body row'>
+        <div class='control-panel-grouping col-sm-offset-1 col-sm-10'>
+          {{#lg-file-upload store=store action="addImage" as |uploads|}}
+            <div class="uploads">
+              {{#each uploads as |upload|}}
+                {{#unless upload.isDeleted}}
+                  <div class="row">
+                    <div class="col-md-4">
+                      <h4>Uploading</h4>
+                      <h3>{{upload.name}}</h3>
+                    </div>
+                    <div class="col-md-2">
+                      <h4>Preview</h4>
+                      {{lg-file-preview file=upload.file previewQueue=imagePreviewQueue}}
+                    </div>
+                    <div class="col-md-6">
+                      <h4>Upload Progress</h4>
 
-    {{#lg-file-upload store=store action="addImage" as |uploads|}}
-      <div class="uploads">
-        {{#each uploads as |upload|}}
-          {{#unless upload.isDeleted}}
-            <div class="row">
-              <div class="col-md-4">
-                <h4>Uploading</h4>
-                <h3>{{upload.name}}</h3>
-              </div>
-              <div class="col-md-2">
-                <h4>Preview</h4>
-                {{lg-file-preview file=upload.file previewQueue=imagePreviewQueue}}
-              </div>
-              <div class="col-md-6">
-                <h4>Upload Progress</h4>
+                      <p>
+                        Uploaded {{bytes-to-human upload.uploadedBytes}} of
+                        {{bytes-to-human upload.totalBytes}}.
+                        Rate: {{bytes-to-human upload.bitRate}}/second.
+                      </p>
 
-                <p>
-                  Uploaded {{bytes-to-human upload.uploadedBytes}} of
-                  {{bytes-to-human upload.totalBytes}}.
-                  Rate: {{bytes-to-human upload.bitRate}}/second.
-                </p>
-
-                {{#lg-progress-bar percent=upload.percentComplete as |humanPercent|}}
-                  {{round-number humanPercent}}%
-                {{/lg-progress-bar}}
-              </div>
+                      {{#lg-progress-bar percent=upload.percentComplete as |humanPercent|}}
+                        {{round-number humanPercent}}%
+                      {{/lg-progress-bar}}
+                    </div>
+                  </div>
+                {{/unless}}
+              {{/each}}
             </div>
-          {{/unless}}
-        {{/each}}
+          {{/lg-file-upload}}
+        </div>
       </div>
-    {{/lg-file-upload}}
+    </div>
   </div>
+
+  {{#if notCreatingNewImageSet}}
+    <div class='col-sm-3'>
+      <div class='cv-request-control-panel'>
+        <div class='control-panel-header'>
+          Manage CV Requests
+        </div>
+        <div class='contorl-panel-body row'>
+
+          {{#if model.id}}
+            {{#if model.cvRequestPending}}
```

Comment:
Nvm, I see a second conditional below where you show a 'Request CV' button that shouldn't be there if the model is not saved. using a more semantic property like `{{#if model.isSaved}}` would be better — ember-data has this, not sure if that's the right prop name though.